specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/azureWebCategory.json

```diff
@@ -0,0 +1,184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2020-07-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/azureWebCategories/{name}": {
+      "get": {
+        "tags": [
+          "AzureWebCategories"
+        ],
+        "operationId": "AzureWebCategories_Get",
```

Comment:
Probably don't want the keyword 'Azure' in the operation group (the first part of the operation id).  This will be used to name classes and methods/properties in generated SDKs, and having 'Azure' as a prefix is likely redundant.

Reply:
The same is true for the Tag

Reply:
Please apply throughout these methods, as this should be consistent

Reply:
We call firewall resource as azureFirewalls. We are using the same naming standard
Please see \specification\network\resource-manager\Microsoft.Network\stable\2020-07-01\azureFirewall.json
This is too late for us to change as we have manifest dependencies